docs/guide/machine-learning.asciidoc

```diff
@@ -1,9 +1,13 @@
 [[machine-learning]]
 == Machine Learning
 
+[discrete]
+[[ml-trained-models]]
+=== Trained models
+
 Eland allows transforming trained models from scikit-learn, XGBoost,
```

Comment:
> Eland allows transforming trained models from scikit-learn, XGBoost,

Feel free to ignore this comment if you just want to add the new content, but this sentence is a little unclear to me (both here and in the readme). Does it mean:

1. Eland enables you to transform trained xxx models from yyy into serialized (what?), which can then be used as inference models in Elasticsearch.
2. Eland enables you to transform and import xxx models from yyy into your Elasticsearch cluster.